doc/content/htgr/htr-10/griffin_htr10_results.md

```diff
@@ -0,0 +1,129 @@
+# HTR-10 Griffin Neutronics Results
+
+*Contact: Javier Ortensi, Javier.Ortensi@inl.gov*
+
+This section is a short summary of the results section of [!citep](HTR-10Benchmark)
+and looks at the various results carried out by Griffin for various HTR-10 calculations.
+The two main input files are the *htr-10-critical.i* and the *htr-10-full.i* which are
+documented in detail in the Griffin Neutronics Model Section.
+
+
+## Running an Input File
+
+If the user has access to Griffin built upon a working Moose Framework the following
+command from the shell prompt will execute the input file.
+
+```language=bash
+
+/projects/griffin/griffin-opt -i htr-10-critical.i
+
+```
+
+## *Initial Critical* Core Results
+
+The first set of results for this model deal with the *htr-10-critical.i* input file
+which is the initial criticality benchmark. [critical_eigen_values] summarizes the
+results of Griffin using tensor diffusion coefficients (TDC) and super homogenization (SPH)
+compared against MCNP and Serpent using ENDFB-VI and Serpent using ENDFB-VII.r1.
+As can be seen in the table below, Griffin matches with the accepted Serpent solution very well.
+
+!table id=critical_eigen_values caption=Eigenvalues computed with different codes for the initial critical configuration. (MCNP results obtained from [!citep](IRPhEP))
+| Code  | $k_eff$  | uncertainty rel. error (pcm)  |
+| :- | :- | :- | :- |
+| MCNP (ENDFB-VI)  | 1.01190 | +/- 21 |
+| Serpent (ENDFB-VI) | 1.01025 | +/- 5.1 |
+| Serpent (ENDFB-VII.r1) | 1.00023 | +/- 2.3 |
+| Griffin TDC-SPH-Diffusion | 1.00089 | 67.3 |
```

Comment:
this is odd. We should get perfect preservation with SPH

Reply:
From the report 
https://www.osti.gov/biblio/1468643-benchmark-analysis-htr-mammoth-reactor-physics-application